docs/compatibility.md

```diff
@@ -0,0 +1,5 @@
+## Compatibility of MMCV
+
+### MMCV v1.3.10
+
+In order to flexibly support more backends and hardwares like `NVIDIA GPUs` and `AMD GPUs`, the directory of `mmcv/ops/csrc` is refactored. Note that this refactoring will not affect the usage in API. For related information, please refer to [PR1206](https://github.com/open-mmlab/mmcv/pull/1206).
```

Comment:
Need to tell how it is refactored, e.g., what is the change before and after refactoring.